3.9/debian-10/rootfs/opt/bitnami/scripts/librabbitmq.sh

```diff
@@ -755,7 +755,8 @@ rabbitmq_initialize() {
         am_i_root && chown -R "$RABBITMQ_DAEMON_USER:$RABBITMQ_DAEMON_GROUP" "$dir"
     done
 
-    if ! is_mounted_dir_empty "$RABBITMQ_DATA_DIR"; then
+    # use realpath to avoid symlink issues
```

Comment:
Could you add some more info like the issue ID to the comment? e.g.:
```suggestion
    # use realpath to avoid symlink issues (#184)
```